nltk/grammar.py

```diff
@@ -734,114 +735,456 @@ def is_chomsky_normal_form(self):
         """
         return self.is_flexible_chomsky_normal_form() and self._all_unary_are_lexical
 
-    def chomsky_normal_form(self, new_token_padding="@$@", flexible=False):
+    ##################################################
+    # Stefan Kaufmann's proposed changes
```

Comment:
@stefkauf can we drop such comments please, and just identify you as a co-author at the top of the module?